README.md

```diff
@@ -1 +1,197 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<!-- <details>
+  <summary>REACT JS</summary>
+  <ul>
+    <li><a href="https://es.react.dev/">REACT JS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>REDUX JS</summary>
+  <ul>
+    <li><a href="https://redux.js.org/">REDUX JS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>JavaScript</summary>
+  <ul>
+    <li><a href="https://lenguajejs.com/javascript/">JavaScript</a></li>
+  </ul>
+</details> -->
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/es/">Ruby</a></li>
+  </ul>
+</details>
+
+
+## Key Features
+- Implement classes and objects in Ruby.
+- Implement encapsulation and inheritance with Ruby.
+- Run a program using the command line.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## LIVE DEMO 
+
+ 🚀 Live Demo
+
+To see the display of my portfolio use this link: (https://budget-app-rod.onrender.com/) 
+
+
+You can also see a video of mine of a demonstration in this link (https://www.youtube.com/watch?v=D0-xjcWNNj4)
+
+<!-- GETTING STARTED -->
+
+
+## 💻 Getting Started 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+* ruby
+* any code editor but mostly use Vs Code and Sublime Text.
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Blog-App-Rod
+
+  git clone https://github.com/rodrigopenayo98/Budget-app-rod.git
+
+```
+
+<!-- ### Install
+
+Install this project with:
+
+```sh
+ npm install
+``` -->
+
+<!-- ### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run start
+``` -->
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rubocop -A
+```
+
+### Run Linter Tests
+
+To run linter tests, run the following command:
+
+```sh
+  rubocop -A
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Rodrigo Penayo**
+
+- GitHub: [@rodrigopenayo98](https://github.com/rodrigopenayo98)
+- Twitter: [@rodrigopenayo98](https://twitter.com/rodrigopenayo98)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rodrigo-penayo-391226158/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Improve user interface**
+- **Create a mobile version**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/rodrigopenayo98/Budget-app-rod/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+- We would like to thank Microverse community.
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=of%20the%20design-,requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author,-.%20Therefore%2C%20you%20must), Please gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)

Reply:
Done! ✅